Otus-DevOps-2018-02/ifqthenp_microservices
# HW 26. Monitoring and Logging with Kubernetes (kubernetes-5)

# Completed Homework №


## The following tasks have been completed

- :large_blue_diamond: Installed Prometheus using Helm
- :large_blue_diamond: Configured Prometheus to monitor metrics for Reddit app services
- :large_blue_diamond: Installed Grafana using Helm
- :large_blue_diamond: Used previously created dashboards in Grafana for application monitoring

## How to get project running

- Follow instructions in homework slides

## How to check that project actually works

- Ensure that the build passed TravisCI tests
  - [#andrei_bogomja](https://devops-team-otus.slack.com/messages/C9M6L1XB3) Slack channel
  - actual GitHub pull request page
- Type URL in browser address bar `http://reddit-prometheus` and `http://reddit-grafana`

## PR checklist
